style.css

```diff
@@ -1,136 +1,450 @@
-  flex-direction: row;
+  flex-direction: column;
   justify-content: center;
-  align-items: center;
-  gap: 9em;
-  background-color: #3c3a39;
   width: auto;
-  height: 2.5em;
 }
 
-/*  user name at the menu  */
-a {
+/* The navigation selector */
+nav {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 1.8rem 1.8rem 1rem 1.8rem;
+  background: var(--navcolor);
+}
+
+nav > a {
   text-decoration: none;
-  color: #fff;
+  color: var(--txtcolor);
+  font-family: 'Roboto', sans-serif;
+  font-size: 1rem;
+  font-weight: 700;
+}
+
+/* headline selector */
+#headline {
+  display: flex;
+  flex-direction: column;
+  height: auto;
+  background: var(--bkcolor);
+  padding: 6rem 2rem 2rem 2rem;
+  background-image: url(Resources/Images/illustration-header-mobile.png);
+  background-repeat: no-repeat;
+  background-position: right 19rem;
+  background-size: 7rem;
+}
+
+#headline > h1 {
+  color: var(--maincolor);
+  font-family: 'Crete Round', serif;
+  font-weight: 400;
+  line-height: 3.5rem;
+  font-size: 3rem;
+  margin-top: 4rem;
+}
+
+#headline > h1 > span {
+  display: block;
 }
 
-.name_div {
+#headline > h2 {
   font-family: 'Roboto', sans-serif;
-  font-style: normal;
   font-weight: 700;
-  font-size: 105%;
-  line-height: 100%;
-  color: #fff;
+  color: var(--txtcolor);
+  padding-top: 1rem;
 }
 
-/* the mobile menu icon */
-.menu_icon {
-  height: 1.5em;
-  width: 8%;
+#headline > p {
+  color: var(--txtcolor);
+  padding-top: 1rem;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 24px;
+  width: 20rem;
+  flex-grow: 0;
+  margin: 0 0;
 }
 
-/*  headlines contaner  */
-#headlines {
+.footericon > ul {
+  list-style-type: none;
   display: flex;
-  height: 39.5em;
-  width: auto;
-  border-radius: 0;
-  background: #1c1a19;
-  background-image: url("Resources/Images/illustration-header-mobile.png");
-  background-repeat: no-repeat;
-  background-size: 40%;
-  background-position: 100% 120%;
+  flex-direction: row;
+  justify-content: space-around;
+  width: 10rem;
+  padding-top: 2rem;
+  padding-bottom: 0.5rem;
+  padding-left: 0.5rem;
 }
 
-/*  headlines test  */
-.headline_texts {
-  height: 60%;
-  width: 85.5%;
-  margin: 7%;
-  position: relative;
-  top: 20%;
+#headline > ul {
+  list-style-type: none;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  width: 10rem;
+  padding-top: 2rem;
+  padding-bottom: 3.5rem;
+  padding-left: 0.5rem;
+}
+
+#headline > ul > li {
+  align-items: center;
 }
 
-/*  social media links  */
-.social_media {
+/* work selector */
+#work {
   display: flex;
-  flex-direction: row;
+  flex-direction: column;
   justify-content: center;
-  align-items: center;
-  height: 11%;
-  width: 60%;
-  gap: 1ch;
-  list-style: none;
+  height: auto;
+  background: var(--txtcolor);
+  color: var(--txtcolor2);
+  padding: 0 2rem 3rem 2rem;
 }
```

Comment:
- [ ] It is required that the main container of the section has a `display: grid;` property. Kindly change it from flex to grid

Reply:
done